engine/src/main/java/org/terasology/persistence/serializers/WorldSerializerImpl.java

```diff
@@ -73,7 +73,11 @@ public WorldSerializerImpl(EngineEntityManager entityManager, PrefabSerializer p
         }
 
         for (Prefab prefab : prefabManager.listPrefabs()) {
-            world.addPrefab(prefabSerializer.serialize(prefab));
+            try {
+                world.addPrefab(prefabSerializer.serialize(prefab));
+            } catch (Throwable ex) {
```

Comment:
Isn't it a bad practice to catch `Throwable` ? Something more specific to the serialization failure we can catch, like the stack overflow directly?